vtm-gdx-poi3d/src/org/oscim/gdx/poi3d/ModelPosition.java

```diff
@@ -37,9 +38,14 @@ public float getRotation() {
         return rotation;
     }
 
-    public void setPosition(double lat, double lon, float rotation) {
+    public float getXRotation() {
+        return xRotation;
+    }
+
+    public void setPosition(double lat, double lon, float rotation, float xRotation) {
         this.y = MercatorProjection.latitudeToY(lat);
         this.x = MercatorProjection.longitudeToX(lon);
         this.rotation = rotation;
+        this.xRotation = xRotation;
```

Comment:
Sorry that I barge in here.
We usually don't need xRotation. Please rotate model while export:
https://github.com/libgdx/libgdx/wiki/Importing-Blender-models-in-LibGDX#setting-the-coordinate-system-up-axis

Reply:
> Sorry that I barge in here.

Please everyone do, more eyes are welcome. :slightly_smiling_face: 
Purpose remains always stability + performance.

Reply:
I'm not sure if it's solved, my Blender2.79 + fbx-conv is causing the z-up building models to be mowing down. This might be the reason (fbx-conv or blender change?) 
https://github.com/libgdx/fbx-conv/issues/102

By the way, horizontal inverse problem has moved to #686

Reply:
Seems that fbx-conf is responsible for those results.